### PR TITLE
Minor Enhancements to README

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
-Repo for the [DJS turorials](https://www.youtube.com/watch?v=UcLspwognk0&list=PLWnw41ah3I4ZfNLV3by7nB6JO2WcCc3Wj) on [MenuDocs](https://www.youtube.com/channel/UCpGGFqJP9vYvzFudqnQ-6IA)
+Repository for the [Discord.js Turorials](https://www.youtube.com/watch?v=UcLspwognk0&list=PLWnw41ah3I4ZfNLV3by7nB6JO2WcCc3Wj) series on [MenuDocs](https://www.youtube.com/channel/UCpGGFqJP9vYvzFudqnQ-6IA).
 
-[![Discord server](https://discordapp.com/api/guilds/416512197590777857/embed.png?style=banner4)](https://discord.gg/MgVaazZ)
+[![Discord Server](https://discordapp.com/api/guilds/416512197590777857/embed.png?style=banner4)](https://discord.gg/MgVaazZ)
+
+## Links
+- [YouTube](https://www.youtube.com/channel/UCpGGFqJP9vYvzFudqnQ-6IA)
+- [Discord.js Series](https://www.youtube.com/watch?v=UcLspwognk0&list=PLWnw41ah3I4ZfNLV3by7nB6JO2WcCc3Wj)
+- [Discord Server](https://discord.gg/MgVaazZ)
```
Comment (duncte123) [inline README.md:30]: Why the invite 3 times?

Comment (Strandxo) [reply README.md:30]: With these minor enhancements, just change it to major enhancements, I'm talking links to good resources, links to certain MenuDocs discord.js tutorials etc.

**Comment by ghost** [comment]
@Strandxo, I've made some changes, so please review this pull-request.

**Comment by Strandxo** [comment]
I like the idea of displaying how to surf through the branches however, if it can be displayed all in a singular image that would be perfect then ill merge the request.

**Pull request closed without merging**